Sources/TensorFlow/Operators/Math.swift

```diff
@@ -2158,6 +2158,36 @@ internal func _vjpMatmul<Scalar: TensorFlowFloatingPoint>(
     })
 }
 
+public extension Tensor where Scalar: TensorFlowFloatingPoint {
+    /// Computes the QR decomposition of each inner matrix in `tensor` such that
```

Comment:
```suggestion
    /// Returns the QR decomposition of each inner matrix in the tensor such that
```
- #160
- `tensor` is an undefined identifier. Use "the tensor" instead.